ui/app/mainui/activitycenter/views/ActivityNotificationBase.qml

```diff
@@ -20,6 +20,8 @@ Item {
     property alias ctaComponent: ctaLoader.sourceComponent
     property alias previousNotificationIndex: dateGroupLabel.previousMessageIndex
 
+    signal activityCenterClose()
```

Comment:
```suggestion
    signal activityCenterClosed()
```

Reply:
Mb `closeActivityCenter`? The handler of this signal should close activity center popup, not emitor

Reply:
Isn't it now called Notification Center? :smile:

Reply:
> Isn't it now called Notification Center? 😄

No, we still have activity center with activity center notifications and notifications with system notifications